### DISCUSSION TITLE
Windows 10 start menu feels choppy

### DISCUSSION THREAD

**Discussion by justin13601** [valinet/ExplorerPatcher#1491] [Q&A]
I'm using EP to get the Windows 10 start menu; however, it sort of feels choppy when I open it. It seems the dock instantly appears rather than flow in as in original Windows 10. Is this just an unavoidable characteristic, or something to do with settings/drivers?

Some of the Windows 11 flyouts also seem laggy/choppy, would this be due to outdated drivers or simply how Windows 11 operates?

**Comment by pyrates999** [comment]
Check for driver updates for your GPU.
